.gitmodules

```diff
@@ -4,6 +4,9 @@
 [submodule "rabit"]
 	path = rabit
 	url = https://github.com/dmlc/rabit
+[submodule "nccl"]
+	path = nccl
+	url = https://github.com/h2oai/nccl
```

Comment:
I see, I have created a forked repo in https://github.com/dmlc/nccl, let us push changes to here before NVidia's changes are reflected and we redirect to nvidia/nccl